lib/game.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+require 'date'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at, :author
+
+  def initialize(publish_date, multiplayer, last_played_at, id = nil)
+    super(publish_date, id)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+    @author = author
+  end
+
```

Comment:
As per the UML class diagram, while creating a game, it should ask for `Genre`, `Author`, and `Label` details. However, in your case, it is only asking for the Author. Kindly fix this issue.